### ISSUE TITLE
Document the origin of the m2ee name

### ISSUE THREAD

**Issue by knorrie** [mendix/m2ee-tools#24]
...since it's an interesting story, and I suspect some users sometimes wonder what this weird name means.

**Comment by hevisko** [comment]
YEs, we'd like to know :)

**Comment by MarkvanMents** [comment]
Perhaps we can have it as a blog post - I'll link to it from my documentation.

**Comment by hevisko** [comment]
yeah, it seems to be a very interesting story that we are kept in the dark for :(=)
